addons/mail/static/src/models/message_reaction_group_item.js

```diff
@@ -0,0 +1,31 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one} from '@mail/model/model_field';
+
+registerModel({
+    name: 'MessageReactionGroupItem',
+    recordMethods: {
+        /**
+         * Handles click on the reaction group Item in massage context dialog,
+         *in order to set messageContextState in messageContextViewOwner.
+         * @param {MouseEvent} ev
+         */
+        onClickReactionGroupItem(reactionGroup) {
+            this.messageContextViewOwner.update({ messageContextState: reactionGroup });
+        },
```

Comment:
As said in a previous comment:
Just name it `onClick`, and use `this.messageReactionGroup` field instead.